Dockerfile.uiforge

```
@@ -0,0 +1,41 @@
+RUN cp -r src/assets dist/assets
+
+# ── Stage 2: Runtime with mcpgateway.translate ───────────────
+FROM python:3.12-slim
+
+RUN apt-get update -qq && \
+    apt-get install -y --no-install-recommends curl ca-certificates && \
+    curl -fsSL https://deb.nodesource.com/setup_22.x | bash - && \
+    apt-get install -y --no-install-recommends nodejs && \
+    apt-get clean && rm -rf /var/lib/apt/lists/*
+
+RUN pip install --no-cache-dir mcp-contextforge-gateway
```

Hadolint (2.14.0)

[warning] Pin versions in pip. Instead of `pip install <package>` use `pip install <package>==<version>` or `pip install --requirement <requirements file>`

(DL3013)